annual-reviews-author-date.csl

```diff
@@ -224,16 +260,43 @@
             <number variable="edition" form="ordinal"/>
             <label variable="edition" form="short"/>
           </group>
+          <text macro="status"/>
         </group>
       </else-if>
       <else-if type="article-journal" match="any">
         <text macro="container-title"/>
         <text macro="locator"/>
       </else-if>
+      <else-if type="paper-conference" match="any">
+        <choose>
+          <if variable="volume" match="any">
+            <text macro="container-title"/>
+            <text macro="locator"/>            
+          </if>
+          <else>
+            <text macro="container-title"/>
+            <text macro="locator"/>
+            <choose>
+              <if variable="publisher" match="any">
+                <group delimiter=". " prefix=". ">
+                  <text macro="publisher"/>
+                  <text macro="status"/>
```

Comment:
this looks like we could get status twice -- once from the locator macro, one here. Am I missing something?